ansible/playbook.yml

```
@@ -40,3 +40,4 @@
     - mongodb_query_exporter
     - ingress
     - post_deployment
+    - tldraw-client-core
```

the tldraw-client-core ansible role must bevor the post_deployment role.
The post_deployment ansible role mark the end from the deployment run
```suggestion
    - mongodb_query_exporter
    - tldraw-client-core
    - ingress
    - post_deployment
```